Dockerfile.php5.6-busybox

```diff
@@ -0,0 +1,127 @@
+FROM ubuntu:xenial as builder
+
+ENV PHP_VERSION 5.6.31
```

Comment:
Should this be an ARG instead? It would allow us to pass it in at the command line without having to change other things?